expression/builtin_cast_vec_test.go

```diff
@@ -61,6 +61,7 @@ var vecBuiltinCastCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETJson, childrenTypes: []types.EvalType{types.ETJson}},
 		{retEvalType: types.ETJson, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&jsonStringGener{}}},
 		{retEvalType: types.ETJson, childrenTypes: []types.EvalType{types.ETDecimal}},
+		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETJson}},
```

Comment:
Do we need to introduce a data generator that can generate datetime-format JSON values? refer [this](https://github.com/pingcap/tidb/blob/master/expression/bench_test.go#L501-L509)

Reply:
I have introduce a data generator that can generate datetime-format JSON values. Is this right?